src/Mod/Draft/DraftVecUtils.py

```diff
@@ -602,7 +602,8 @@ def find(vector, vlist):
     return None
 
 
-def closest(vector, vlist):
+def closest(vector, vlist, returnLength=False):
```

Comment:
Please use proper Pythonic style for new code; for variables, this means `snake_case`; `return_length=False`.